### ISSUE TITLE
Directly send the link if already synced with Nextcloud

### ISSUE THREAD

**Issue by biva** [nextcloud/nextcloud-filelink#52]
## Feature request

Today, if I want to send a document that is already synced on NC, it copies it in the MailAttachment folder, and then sends the link. 

It should recognize if the file is already in a folder synchronized by Nextcloud, and only create and send the link, without copying the file in the MailAttachment folder.

**Comment by creopard** [comment]
@biva: if you already have the Nextcloud client at hand you could also generate the link there and just paste it in Thunderbird. Thats even faster and optionally password protected.

I could imagine that such a mighty feature request within the filelink-addon would be rather complex, slow and prone to errors.

**Comment by biva** [comment]
@creopard Yes, this is what I do now, but this represents 7 or 8 clicks, and to add `/download` at the end of the link :( While moving a file to the message editor is only 2 clicks.

**Comment by biva** [comment]
Hi here, Thunderbird just announced compatibility with WeTransfer: https://blog.mozilla.org/thunderbird/2019/05/wetransfer-file-transfer-now-available-in-thunderbird/

It made me think again to this topic. Do you think we could go towards a solution? Thanks!

**Comment by biva** [comment]
I see that it's now possible to "share by mail" (https://github.com/nextcloud/desktop/issues/1180) within the desktop app. It's great, but it's not as convenient as a sharing within Thunderbird: indeed, it sends a dedicated email, it's not possible to edit the email before and the link cannot be included in an email I write.

It would be great to have this feature: what do you think?
